Crash checking publiccode.yml from Azure Devops

```
pcvalidate https://dev.azure.com/atsmi-open/_git/Controllo%20di%20Gestione\?path\=%2Fpubliccode.yml
panic: runtime error: invalid memory address or nil pointer dereference        
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x6708f6]        goroutine 1 [running]:
github.com/alranel/go-vcsurl.IsFile(0x0, 0xc0000e5e78)
        /home/surf/go/src/github.com/alranel/go-vcsurl/vcsurl.go:90 +0x26
github.com/alranel/go-vcsurl.GetRawRoot(0x0, 0x0)
        /home/surf/go/src/github.com/alranel/go-vcsurl/vcsurl.go:170 +0x2f
main.main()
        /home/surf/go/src/github.com/italia/publiccode-parser-go/pcvalidate/pcvalidate.go:43 +0x610
```

--- a/pcvalidate/pcvalidate.go
+++ b/pcvalidate/pcvalidate.go
@@ -62,12 +62,16 @@ func main() {
 	var err error
 	if ok, url := isValidURL(flag.Args()[0]); ok {
 		// supplied argument looks like an URL
-		url = vcsurl.GetRawFile(url)
+		rawUrl := vcsurl.GetRawFile(url)
+		if rawUrl == nil {
+			fmt.Fprintf(os.Stderr, "Code hosting provider not supported for %s\n", url)
+			os.Exit(1)
+		}
 		if p.RemoteBaseURL == "" {
-			p.RemoteBaseURL = vcsurl.GetRawRoot(url).String()
+			p.RemoteBaseURL = vcsurl.GetRawRoot(rawUrl).String()
 		}
 
-		err = p.ParseRemoteFile(url.String())
+		err = p.ParseRemoteFile(rawUrl.String())
 	} else {
 		// supplied argument looks like a file path
 		err = p.ParseFile(flag.Args()[0])

